docker-compose.yml

```diff
@@ -48,14 +48,21 @@ services:
       - spotlight
 
   solr:
-    image: bitnami/solr:8.11.1
-    env_file:
-      - .env
-    ports:
-      - 8983:8983
+    environment:
+      - SOLR_PORT # Set via environment variable or use default defined in .env file
+      - SOLR_VERSION # Set via environment variable or use default defined in .env file
+    image: "solr:${SOLR_VERSION}"
     volumes:
-      - spotlight_solr:/bitnami/solr
-      - ./lib/generators/spotlight/templates/solr/config:/opt/custom/spotlightset
+      - $PWD/lib/generators/spotlight/templates/solr/conf:/opt/solr/conf
+    ports:
+      - "${SOLR_PORT}:8983"
+    entrypoint:
+      - docker-entrypoint.sh
+      - solr-precreate
+      - blacklight-core
+      - /opt/solr/conf
+      - "-Xms256m"
+      - "-Xmx512m"
```

Comment:
why the JVM memory settings? I usually don't bother in docker unless something is breaking

Reply:
I just copied from blacklight: https://github.com/projectblacklight/blacklight/blob/main/docker-compose.yml#L33-L39  Want me to remove this?

Reply:
yeah, I think unless we need it better not to set anything.

Reply:
Okay, they've been removed.

Reply:
thank you!